rally_ovs/plugins/ovs/ovsclients_impl.py

```diff
@@ -282,6 +282,26 @@ def port_group_del(self, port_group):
             params = [port_group]
             self.run("pg-del", [], params)
 
+        def lb_add(self, lb_name, lb_vip, lb_proto):
+            params = [lb_name, lb_vip, '\'\'', lb_proto]
+            self.run("lb-add", [], params)
+        
+        def lb_set_vip_backends(self, lb_name, lb_vip, lb_backends):
+            vips_set='vips=\'\"{}\"=\"{}\"\''.format(lb_vip, lb_backends)
```

Comment:
Nit, you can get rid of the `\` before the double quotes. It might make the line a bit easier to read.

Reply:
Ack, thanks.